diff -Nur gwenview-1.2.0-orig/admin/ltmain.sh gwenview-1.2.0/admin/ltmain.sh
--- gwenview-1.2.0-orig/admin/ltmain.sh	2004-12-19 18:02:44.000000000 +0100
+++ gwenview-1.2.0/admin/ltmain.sh	2005-06-02 01:04:14.000000000 +0200
@@ -2040,6 +2040,8 @@
 	*/* | *\\*) . $lib ;;
 	*) . ./$lib ;;
 	esac
+	# Temporary work-around for missing %{_libdir}/libfam.la
+	test -n "$dependency_libs" && dependency_libs=$(echo $dependency_libs | sed 's![^ ]*libfam.la!!')
 
 	case $host in
 	*-*-darwin*)
